src/blueapi/core/bluesky_types.py

```diff
@@ -1,8 +1,11 @@
 import inspect
 from collections.abc import Callable, Mapping
+from collections.abc import Generator as ConcreteGenerator
```

Comment:
What is `Generator` colliding with? I think it would be less confusing to use `abc.Generator` instead of renaming it to something custom.

Reply:
Forgot to change that back, my bad